test/cpp/interop/client_helper.cc

```diff
@@ -107,14 +97,10 @@ std::shared_ptr<Channel> CreateChannelForTestCase(
   snprintf(host_port, host_port_buf_size, "%s:%d", FLAGS_server_host.c_str(),
            FLAGS_server_port);
 
-  if (test_case == "service_account_creds") {
```

Comment:
Also remove the places in interop_client.h interop_client.cc and client.cc related to service_account_creds as a test case.